resources/com/boxboat/jenkins/config.example.yaml

```diff
@@ -42,6 +42,12 @@ vaultMap:
     secretIdCredential: vault-secret-id
     tokenCredential: vault-token
     url: http://localhost:8200
+awsProfileMap:
+  default:
+    region: us-east-1
+    accessKeyIdCredential: aws-access-key-id
```

Comment:
should we use a user/pass credential here?  Access key ID is the User and Secret Access Key is the pass?

Reply:
I think that for Vault we do not use user/pass currently where we could with APP_ROLE_ID and APP_ROLE_SECRET_ID.  I believe those are separate secret text credentials

Reply:
For AWS API authentication Access Key and Secret Key are the only authentication options (besides EC instance profile). So we will need to create a jenkins IAM user and generate access keys.

Reply:
Updating naming scheme to `accessKeyIdCredential` and `secretAccessKeyCredential`
Updated `dockcmd` to use:
`${AWS_ACCESS_KEY_ID}`
`${AWS_SECRET_ACCESS_KEY}`